benchmarks/suites/polybench/durbin/durbin.cxx

```diff
@@ -35,10 +35,15 @@ static void kernel(pbsize_t n,
 void run(State &state, pbsize_t pbsize) {
   pbsize_t n = pbsize; // 2000
 
-  auto r = state.allocate_array<real>({n}, /*fakedata*/ true, /*verify*/ false, "r");
+  auto r = state.allocate_array<real>({n}, /*fakedata*/ false, /*verify*/ false, "r");
   auto y = state.allocate_array<real>({n}, /*fakedata*/ false, /*verify*/ true, "y");
   auto z = state.allocate_array<real>({n}, /*fakedata*/ false, /*verify*/ false, "z");
 
-  for (auto &&_ : state)
-    kernel(n, r, y, z);
+  for (auto &&_ : state.manual()) {
+    initialize_input_vector(n, r);
```

Comment:
As discussed, the initialization needs to be just once, not during every iteration, aśit does not change